pkg/controller/complianceremediation/complianceremediation_controller.go

```diff
@@ -369,6 +381,115 @@ func (r *ReconcileComplianceRemediation) countKubeUnmetDependencies(rem *compv1a
 	return nMissingDeps, nil
 }
 
+func (r *ReconcileComplianceRemediation) handleValueRequired(rem *compv1alpha1.ComplianceRemediation, logger logr.Logger) (reconcile.Result, error) {
+	rCopy := rem.DeepCopy()
```

Comment:
Just a point of optimization wrt DeepCopy; it would be better to make the DeepCopy closer to the call to Update. Ideally under the same condition block, so a no-op run of the function won't make the copy.